app/addons/setup/__tests__/optionalSettings-test.js

```diff
@@ -0,0 +1,48 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+import utils from "../../../../test/mocha/testUtils";
+import React from "react";
+import ReactDOM from "react-dom";
+import OptionalSettings from '../components/OptionalSettings';
+import sinon from "sinon";
+import {mount} from 'enzyme';
+
+const assert = utils.assert;
+
+//this was commented out. I imagine it needs to be updated
+describe.skip('Setup Components', () => {
```

Comment:
Should we update these tests or just delete them?

Reply:
I don't really know how to test if the store is updated

Reply:
Ok then remove these tests.